src/java/org/apache/cassandra/tools/nodetool/ListEndpointsPendingHints.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.tools.nodetool;
+
+import java.net.UnknownHostException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import io.airlift.airline.Command;
+import org.apache.cassandra.hints.PendingHintsInfo;
+import org.apache.cassandra.locator.EndpointSnitchInfoMBean;
+import org.apache.cassandra.locator.InetAddressAndPort;
+import org.apache.cassandra.tools.NodeProbe;
+import org.apache.cassandra.tools.NodeTool;
+import org.apache.cassandra.tools.nodetool.formatter.TableBuilder;
+
+@Command(name = "listendpointspendinghints", description = "Print all the endpoints that this node has hints for")
+public class ListEndpointsPendingHints extends NodeTool.NodeToolCmd
```

Comment:
I feel once we have a test for the table, probably this one will be easy, but again I'm not yet sure how to create it at the moment.

Reply:
I am not sure you saw the [dtest](https://github.com/apache/cassandra/pull/1232#commitcomment-57238985) added and our [comments](https://github.com/apache/cassandra/pull/1232#commitcomment-57240666) with Aleks on that?

Reply:
Having a dtest sounds good! That's the reason I do not like commenting on commits, it is hard to track them :smile:

Reply:
@azotcsit This is the link to the dtest and I believe it is also posted in the ticket but probably you thought it's dtest test results or something like that linked there.
https://github.com/apache/cassandra-dtest/pull/162/files

Reply:
> Having a dtest sounds good! That's the reason I do not like commenting on commits, it is hard to track them 😄

I am subscribed to every commit where I comment, a bit of spam sometimes but it works so far  :-)